_posts/2020-12-03-solidity-underhanded-contest-winners.md

```diff
@@ -79,7 +79,7 @@ The exploit is very well hidden and makes use of special unicode symbols that ch
 
 Since multi-line comments work in the same way when read from left to right as well as from right to left, the submission is able to terminate the comment "from right to left" and switches into code mode which is then displayed in the "wrong" order. This causes the day and month variables to be swapped unnoticed and the upgrade is performed much earlier than expected.
 
-You can see the direction change in action by selecting [part of line 65](https://github.com/ethereum/solidity-underhanded-contest/blob/master/submissions_2020/submission11_RobertMCForster/contracts/TimelockUpgrade.sol#L65) in the submission.
+You can see (*or not, depending on your editor 😬*) the direction change in action by selecting [part of line 65](https://github.com/ethereum/solidity-underhanded-contest/blob/master/submissions_2020/submission11_RobertMCForster/contracts/TimelockUpgrade.sol#L65) in the submission.
```

Comment:
If you click on the link you will use github's "editor" and that shows it, right? I mean it hides it...